src/components/ActionButton.native.tsx

```diff
@@ -40,5 +41,5 @@ const styles = StyleSheet.create({
     marginVertical: 10,
     width: '50%',
   },
-  title: { fontSize: 18 },
+  title: { fontSize: 2 },
```

Comment:
Is native `fontSize` a different unit than `px`?

Reply:
Is there a way we can unify web and native `fontSize` here so they can specified as the same value in the component and converted at a lower-level?

Reply:
React native uses a special unit called DP which is a size that is relative to the pixel density of the device. It scales things so that they approximately relate on devices with different size screens.

Since we are using a text component to adjust the font size according to user's setting we can set a scale that matches somewhat to the Web sizes. 
There is no global equivalent to em on Web which reacts to inherited font size, so we have to do it somewhat manually with a custom component.

Reply:
Okay, thanks for explaining